slack_bolt/util/payload_utils.py

```diff
@@ -0,0 +1,210 @@
+from typing import Dict, Any, Optional
+
+
+# ------------------------------------------
+# Public Utilities
+# ------------------------------------------
+
+# -------------------
+# Events API
+# -------------------
+
+
+def to_event(payload: Dict[str, Any]) -> Optional[Dict[str, Any]]:
+    return payload["event"] if is_event(payload) else None
+
+
+def to_message(payload: Dict[str, Any]) -> Optional[Dict[str, Any]]:
+    if is_event(payload) and payload["event"]["type"] == "message":
+        return to_event(payload)
+    return None
+
+
+def is_event(payload: Dict[str, Any]) -> bool:
+    return (
+        payload is not None
```

Comment:
It never happens but just for safety to avoid resulting in 500 internal server error with an invalid request body. I always want to reject such with 40x.